motor_control/throttle/variable_control.ino

```diff
@@ -0,0 +1,19 @@
+#include "variable_control.h"
+
+VariableThrottle::VariableThrottle(const int &pin):Throttle(pin){}
+
+int VariableThrottle::compute_motor_value() {
```

Comment:
Hmm we should still plan on having some sort of ramping here.

Reply:
Definitely agreed! Just not using the delay function as we discussed